src/utils/victim-identification/victim.identifier.spec.ts

```diff
@@ -115,6 +109,16 @@ class MockEthersProviderExtended extends MockEthersProvider {
     when(this.getCode).calledWith(address, blockNumber).mockReturnValue(Promise.resolve(code));
     return this;
   }
+
+  public addStorageExtended(
+    contract: string,
+    slot: ethers.BigNumberish,
```

Comment:
Just to make sure, this was implemented just so the parameter of `slot` can have type of `ethers.BigNumberish`, correct?

Want to make sure I'm not missing something else.

Reply:
Correct 👍